.github/workflows/ci.yml

```
@@ -0,0 +1,31 @@
+name: CI
+on:
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.version == 'nightly' }}
+    strategy:
+      matrix:
+        version:
+          - '1'
```

Always good to test on the earliest supported Julia version
```suggestion
          - '1.0'
          - '1'
```